grpc/server.go

```diff
@@ -114,18 +114,18 @@ func RunServer(ctx context.Context, log logging.Logger, grpcServer *grpc.Server,
 	signal.Notify(sigChan, os.Interrupt)
 	go func() {
 		for range sigChan {
-			log.Info("sig received, shutting down PBnJ")
+			log.Info(0, "sig received, shutting down PBnJ")
 			grpcServer.GracefulStop()
 			<-ctx.Done()
 		}
 	}()
 
 	go func() {
 		<-ctx.Done()
-		log.Info("ctx cancelled, shutting down PBnJ")
+		log.Info(0, "ctx cancelled, shutting down PBnJ")
 		grpcServer.GracefulStop()
 	}()
 
-	log.Info("starting PBnJ gRPC server")
+	log.Info(0, "starting PBnJ gRPC server")
```

Comment:
Hey @joelrebel, thanks for tackling this. I'm concerned that this logging signature now doesn't follow the go-logr documentation `log.V(0).Info`. I don't think this allows us to use `go-logr` as is expected/intended. I think this is only necessary because of the way logging is being handled in general here in PBnJ. It's not very good 😞  (I wrote it!). 

What are your thoughts around removing the`packethost/pkg/log/logr` dependency entirely and just using `go-logr`? I apologize. I know it's not what you were wanting to do here and the work to move to `go-logr` touches a lot. I'm willing to do the work to do this update if that's helpful.

Reply:
Yeah, if you wouldn't mind closing this one. I opened https://github.com/tinkerbell/pbnj/pull/116 to do the move to `go-logr` directly.